CAIPs/caip-5.md

```diff
@@ -0,0 +1,71 @@
+---
+caip: 5
+title: The Cosmos interface for CAIP-2
+author: Simon Warta (@webmaster128)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/5, https://github.com/ChainAgnostic/CAIPs/pull/1
+status: Draft
+type: Standard
+created: 2019-12-05
+requires: 2, 6
```

Comment:
Are circular dependencies allowed? Are they strict requirements in both directions here?

Reply:
I think circular dependencies don't harm in this context. It better models the problem than pretensing one interface does not know of the other.

Reply:
I think in this case, it's not really required to make them circular because the use of CAIP-6 to normalize chain_id format is independent IMO of the use-case. In fact, I would add that CAIP-6 could be used to normalize any chain_id regardless of interface

Reply:
Yea - let's please avoid circular dependencies

Reply:
In https://github.com/ChainAgnostic/CAIPs/pull/9/commits/c51894d7e951314d8459803ca105b78dd42b1b94 the two namespaces "cosmos:" and "cosmos-hash:" got merged into one document, avoiding the need for circular dependencies.